apis/dataplane/v1beta1/openstackdataplanenodeset_types.go

```diff
@@ -229,6 +229,7 @@ var ContainerImageDefaults = openstackv1.ContainerImages{
 		CeilometerComputeImage:        getStrPtr("quay.io/podified-antelope-centos9/openstack-telemetry-ceilometer-compute:current-podified"),
 		CeilometerIpmiImage:           getStrPtr("quay.io/podified-antelope-centos9/openstack-telemetry-ceilometer-ipmi:current-podified"),
 		EdpmNodeExporterImage:         getStrPtr("quay.io/podified-antelope-centos9/openstack-telemetry-node-exporter:current-podified"),
+		EdpmKeplerImage:               getStrPtr("quay.io/podified-antelope-centos9/openstack-telemetry-kepler:current-podified"),
```

Comment:
The image doesn't exist (or at least I can't find it). Same goes for the rest of telemetry images above (but that's not part of this PR). Shouldn't we have this fallback default point to an existing destination?

Reply:
Maybe these should have the same values as in config/default/manager_default_images.yaml? This seems to be the case for other services

Reply:
> Maybe these should have the same values as in config/default/manager_default_images.yaml? This seems to be the case for other services

Yeah, I meant to change that for Kepler and the node-exporter image as well. Thanks!

Reply:
The first question I had upon review was why the node-exporter image was changing. I've read this discussion and agree that defaults shouldn't point to an image that doesn't exist, but I'm still left wondering why we look like the odd one out among the other containers in the list.

Why does
quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified   exist, but 
quay.io/podified-antelope-centos9/openstack-telemetry-node-exporter:current-podified  does not?

Reply:
Thank you for the explanation!